web/src/stream_ui_updates.js

```diff
@@ -441,7 +441,7 @@ export function enable_or_disable_add_subscribers_elements(
         const $add_subscribers_button = $container_elem
             .find('button[name="add_subscriber"]')
             .expectOne();
-        $add_subscribers_button.prop("disabled", !enable_elem);
+        $add_subscribers_button.prop("disabled", true);
```

Comment:
I think enable_elem value passed into the function is incorrect. From where we call this function for channel setting, this value should be updated over there.

Reply:
On second thought, even if user can or cannot add subscribers, the `$add_subscribers_button` should be disabled. This seems fair.